peps/pep-0791.rst

```diff
@@ -0,0 +1,158 @@
+PEP: 791
+Title: imath --- module for number-theoretic functions
+Author: Sergey B Kirpichev <skirpichev@gmail.com>
+Sponsor: Victor Stinner <vstinner@python.org>
+Discussions-To: Pending
+Status: Draft
+Type: Standards Track
+Created: 12-May-2025
+Python-Version: 3.15
+Post-History: `12-Jul-2018 <https://mail.python.org/archives/list/python-ideas@python.org/thread/YYJ5YJBJNCVXQWK5K3WSVNMPUSV56LOR/>`__,
+              02-Jun-2019,
```

Comment:
Does it work to add a link to an issue?

```suggestion
              `02-Jun-2019 <https://github.com/python/cpython/issues/81313>`__,
```

Reply:
No.  Linter requires d.p.o or mail.p.o link.

Reply:
Hmm, strictly speaking `Post-History` is the past discussion of _this_ PEP. I think the easiest thing to do is leave the issue out of the header and refer to it somewhere the text if useful.

Reply:
> It doesn't really add much on its own. 

I think it does.  That thread contains opinions of some core developers (involved in development of that module), who not participated in the first thread.

Would it be ok if I reformat the reference to this issue in the text?